MvvmCross.Forms/Platforms/Uap/Core/MvxFormsWindowsSetup.cs

```diff
@@ -41,12 +42,43 @@ public virtual Application FormsApplication
             }
         }
 
+        private List<Assembly> _viewAssemblies;
+        public override IEnumerable<Assembly> GetViewAssemblies()
+        {
+            if (_viewAssemblies == null)
+            {
+                _viewAssemblies = new List<Assembly>(base.GetViewAssemblies());
+            }
+
+            return _viewAssemblies;
+        }
+
+        protected override void InitializeIoC()
+        {
+            base.InitializeIoC();
+            Mvx.RegisterSingleton<IMvxFormsSetup>(this);
+        }
+
+        protected override void InitializeApp(IMvxPluginManager pluginManager, IMvxApplication app)
+        {
+            base.InitializeApp(pluginManager, app);
+            _viewAssemblies.AddRange(GetViewModelAssemblies());
```

Comment:
Why is it necessary for all the viewmodel assemblies to be added into the viewassemblies? If the forms Application class is in the same assembly as the viewmodels, the assembly will already be in the viewassemblies. This override shouldn't be necessary for any platform now

Reply:
This is when you are not using the generic. Maybe we've got to look into cleanup or doing it another way, but that's for another PR.